code/datums/traits/neutral.dm

```diff
@@ -224,3 +224,13 @@
 	SIGNAL_HANDLER
 
 	SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "bad_hair_day", /datum/mood_event/bald)
+
+/datum/quirk/vampire
+	name = "Vampire"
+	desc = "You're a bloodsuckig vampire."
+	value = 0
+	mob_trait = TRAIT_BALD
```

Comment:
Why the fuck is this here, no wonder that one thing was runtiming
```suggestion

```

Reply:
coder

Reply:
project lead